mm2src/coins/watcher_common.rs

```diff
@@ -0,0 +1,37 @@
+use crate::{ValidatePaymentError, INVALID_WATCHER_REWARD_ERR_LOG};
+use mm2_err_handle::prelude::MmError;
+
+pub const REWARD_GAS_AMOUNT: u64 = 70000;
```

Comment:
Also wondering what were the basis on which this amount chosen.

Reply:
This value is used temporarily. It needs to be something a little higher than the amount of gas used when the watchtower spends the maker payment or refunds the taker payment. For that, the contract has to take its' final shape after possible changes and optimizations. Afterwards, we can discuss how much higher than the gas cost of the watchtower do we want to make the reward.

Reply:
Thanks you for the explanation.